packages/generate-html/src/index.ts

```diff
@@ -51,7 +51,21 @@ function languagesStringToArray(languages: string | string[]): string[] {
   return typeof languages === 'string' ? languages.split(',').map((val) => val.trim()) : languages;
 }
 
-const handlebarsBaseOptions = (data: {
+function localizeNumber(lvalue: number | string, locales: string | string[] = 'EN', returnSafeString = true) {
```

Comment:
I added `returnSafeString` to it as originally the `localizeNumber` function you mentioned in your next comment was identical to what I needed here, except that one returned a `SafeString` and the other a normal `string`. I wanted to avoid duplicate code so I moved the function up here instead. But Importing it from pdf-template-helpers and leaving the function signature of `localizeNumber` further down is a better idea, I'll do that.